### ISSUE TITLE
Proposal: naming

### ISSUE THREAD

**Issue by matschaffer** [vaubarth/testingbeat#1]
Pretty sure every other beat follows a (noun)beat pattern so I’d propose this be called testbeat or perhaps something like unitbeat (junitbeat, xunitbeat)

Love the initiative though ❤️

**Comment by vaubarth** [comment]
Indeed, very good point. I considered `Testbeat` in the beginning as well but decided against it since its sounds like a beat thats just a test :-)

`Unitbeat` is really nice though, I will think about other possible names since in the future this will probably read result files from other test frameworks too (not limited to unittest-like output as well)
